bcda/models/models_test.go

```diff
@@ -833,280 +742,6 @@ func (s *ModelsTestSuite) TestGetMaxBeneCount() {
 	assert.EqualError(err, "invalid request type")
 }
 
-func (s *ModelsTestSuite) TestGetBeneficiaries() {
```

Comment:
All of the GetBeneficiary tests are now replaced by tests in service_test.go and repository_test.go.

These tests were verifying three main things.

We can get beneficiary data successfully -> replaced by service_test.go (and updated models_test.go)
We do not retrieve beneficiary data for CCLF files that are not marked as "Complete"
- We replaced this test by using mocks in both service_test.go and repository_test.go
- For service_test.go, we mocked out the repository call go GetCCLFFile such that it only will respond to the call iff the import status of the CCLF is complete (constants.ImportComplete). So if we happened to be using the incorrect status in our query (ex: constants.ImportFail), we would've caught in the service_test because the expectations of the mock were not satisifed.
- For repository.go, we validated that we have the import_status = ? clause in our query for CCLF files.
We do not retrieve beneficiaries that are in the suppressions list. The same approach was done as 2. Combination of service_test and repository_test.go
- For service test, we validated that we are passing in the result of GetSuppressedMBIs into the repository query for GetCCLFBeneficiaries call (by asserting that the mock was called with the correct MBIs). This means that we're passing in the expected suppressed MBIs to the repository call.
- For the repository test, we verify that we have the following condition present iff the caller supplies suppressed (ignored) MBIs. AND ("cclf_beneficiaries"."mbi" NOT IN